akvo/rsr/static/scripts-src/my-results/selectors.js

```diff
@@ -100,29 +100,17 @@ export const getUpdatesChildrenIds = createSelector(
 
 
 export const getPeriodsActualValue = createSelector(
-    /*
-        Return an object on the form:
-        {periodId1: <actualValue1>, periodId2: <actualValue2>,...}
-     */
-    [getPeriodIds, getUpdateObjects, getPeriodsChildrenIds],
-    (periodIds, updateObjects, childUpdateIds) => {
-        return periodIds && updateObjects && !isEmpty(childUpdateIds) && periodIds.reduce((acc, periodId) => {
-            const actualValue = childUpdateIds[periodId].filter(
-                (updateId) => updateObjects[updateId].status == c.UPDATE_STATUS_APPROVED
-            ).reduce(
-                // Actual value is calculated by adding all approved updates with numerical data
-                (acc, updateId) => {
-                    const data = parseInt(updateObjects[updateId].data);
-                    // If data is NaN then data !== data returns true!
-                    if (!(data !== data)) {
-                        return acc + data;
-                    }
-                    return acc;
-                }, 0
-            );
-            return {...acc, [periodId]: actualValue};
-        }, {});
-    }
+  /*
```

Comment:
The problem with this is that period.actual_value isn't updated when we add or change an update. So we'd have to reload the period from the API after posting updates if this is to work.